src/app/utils/getNews.ts

```diff
@@ -0,0 +1,12 @@
+import { schemaNewsResponse } from '@/models/schemes';
+import axios from 'axios';
+
+const getNews = async (keyword: string) => {
+  const response = await axios.get(
```

Comment:
Please use try catch and in the catch block you can return something like that
```
{
	status: ‘error’,
	data: []
}
```
And then when you see an error you’ll be able to show something to user and don’t break the app